### PR TITLE
Initial image

### PR DESCRIPTION
**Pull request by vwbusguy**

### CODE REVIEW

**Comment by ParadoxGuitarist** [comment]
I think we no longer need xeus-python as a req.

**Comment by vwbusguy** [comment]
> I think we no longer need xeus-python as a req.

Good catch.  Removed.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,4 @@
-# Container image source for <course>
-Info here about where to find and run this image.
+# Container image source for ling-110
```
Comment (ParadoxGuitarist) [inline README.md]: ling-102-194

**Review by ParadoxGuitarist — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vwbusguy) [reply README.md]: Updated

**Review by ParadoxGuitarist — APPROVED** [review]

**Pull request merged**